docs/user-guide/flow-computation/continuous-aggregation.md

```diff
@@ -1,8 +1,14 @@
 ---
-description: Provides major use case examples for continuous aggregation in GreptimeDB, including real-time analytics, monitoring, and dashboards. It includes SQL queries and examples to demonstrate how to set up and use continuous aggregation for various scenarios.
+description: Explore continuous aggregation in GreptimeDB for real-time insights. This guide explains how to perform continuous aggregations using the Flow engine, including calculating sums, averages, and other metrics within specified time windows. Learn through examples of real-time analytics, monitoring, and dashboard use cases. Understand how to create source and sink tables, define flows, and write SQL queries for continuous aggregation. Discover how to calculate log statistics, retrieve distinct countries by time window, and monitor sensor data in real-time. This comprehensive guide is essential for developers looking to leverage GreptimeDB's continuous aggregation capabilities for efficient data processing and real-time analytics.
```

Comment:
The description must be around 120~150 characters, so i think it can be refactor:

```
Learn how to use GreptimeDB's continuous aggregation for real-time analytics. Master Flow engine basics, time-window calculations, and SQL queries through practical examples of log analysis and sensor monitoring.
```